Source/Core/Core/HW/ProcessorInterface.cpp

```diff
@@ -108,7 +109,14 @@ void RegisterMMIO(MMIO::Mapping* mmio, u32 base)
                    if ((val & 1) != 0)
                    {
                      GPFifo::ResetGatherPipe();
-                     Fifo::ResetVideoBuffer();
+
+                     // Resetting the FIFO from the CPU thread is dangerous in dual core mode
+                     // As various pointers being used in the GPU thread are reset
+                     // So we'll just let the GPU thread reset it if need be
```

Comment:
Here's how I'd phrase this instead:

> Call Fifo::ResetVideoBuffer() from the video thread. Since that function resets various pointers used by the video thread, we can't call it directly from the CPU thread, so queue a task to do it instead. In single-core mode, AsyncRequests is in passthrough mode, so this will be safely and immediately called on the CPU thread.

Perhaps also add a note that `GPFifo::ResetGatherPipe()` affects CPU state, so it's safe to change it on the CPU thread?